man/rmd-fragments/redirects-configuration.Rmd

```diff
@@ -0,0 +1,11 @@
+If you change the structure of your documentation (vignettes for instance) you can setup redirects.
+One or several now-absent pages can be redirected to a new page (or to a new section of a new page.
```

Comment:
This works by creating a html page that performs a "meta refresh", which isn't the best way of doing a redirect but works everywhere that you might deploy your site.